tst/testlibgap/wscreate.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Small program to test libgap ability to save workspaces.
```

Comment:
While I don't mind adding such a test, I wonder how useful it is in practice; I imagine that for most users of libgap, saving/restoring workspaces will not be usable, because any connections between GAP and GAP objects on the one hand, and client code (such as SageMath) and its "objects" will be lost.

Reply:
It is often useful in practice to load a workspace, as it speeds up starting GAP quite a bit.
This is the primary reason for this stuff by be in Sage's libGAP.

Besides, this test uncovered a real bug in libgap  :-)